spec/support/shared_examples/validators/presence_validator.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples "a presence validator" do |validatable_class, property|
+  # it "includes ValidatesPresence" do
```

Comment:
Another convention we have is not to leave commented out code in the code base.

> I should really write these down somewhere 😳

Reply:
This was an oversight.  It shouldn't have been commented out.

Reply:
Fixed in bcdf764.